NitroxPatcher/Patches/Persistent/Language_LoadLanguageFile_Patch.cs

```diff
@@ -26,6 +26,7 @@ public static void Postfix(string language, Dictionary<string, string> ___string
                 if (!File.Exists(file))
                 {
                     Log.Warn($"No language file was found for at {file}. Using English as fallback");
+                    break;
```

Comment:
`continue` fits here more because you want to terminate the failing language file not all other in the array.

Reply:
I want to stop because no more languages are loading after the first (English)

Reply:
I think that in this case using `break` or `continue` is the same but if you prefer i could change it

Reply:
You are right the outcome is the same but if (for whatever reason) we add more language files to load or just English fails the other file wont be loaded.